subhalo_properties.py

```diff
@@ -593,30 +593,32 @@ def vcom(self) -> unyt.unyt_array:
         return (self.total_mass_fraction[:, None] * self.velocity).sum(axis=0)
 
     @lazy_property
-    def R_vmax(self) -> unyt.unyt_quantity:
+    def R_vmax_unsoft(self) -> unyt.unyt_quantity:
         """
         Radius at which the maximum circular velocity of the halo is reached.
+        No softening is applied to the particle radii.
 
         This includes contributions from all particle types.
         """
         if self.Mtot == 0:
             return None
-        if not hasattr(self, "r_vmax"):
-            self.r_vmax, self.vmax = get_vmax(self.mass, self.radius)
-        return self.r_vmax
+        if not hasattr(self, "r_vmax_unsoft"):
+            self.r_vmax_unsoft, self.vmax_unsoft = get_vmax(self.mass, self.radius, nskip=1)
+        return self.r_vmax_unsoft
 
     @lazy_property
-    def Vmax(self) -> unyt.unyt_quantity:
+    def Vmax_unsoft(self) -> unyt.unyt_quantity:
         """
         Maximum circular velocity of the halo.
+        No softening is applied to the particle radii.
```

Comment:
I would say that all particles retain their original positions, and are not constrained to be at least a softening distance away. Otherwise someone may interpret this as smoothing the particle out using some sort of a kernel?

Reply:
[344dd9a](https://github.com/SWIFTSIM/SOAP/pull/93/commits/344dd9a053a1c5de4ecd50a90f308c5dd1de7aca)